app/views/facility_user_reservations/index.html.haml

```diff
@@ -0,0 +1,66 @@
+= content_for :h1 do
+  = current_facility
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_users", sidenav_tab: "users"
+
+= content_for :tabnav do
+  = render "admin/shared/tabnav_users", secondary_tab: "reservations"
+
+%h1= text("users.reservations.head", name: @user.full_name)
+
+- if @order_details.blank?
+  %p.notice= text("users.reservations.none", name: @user.full_name)
+
+- else
+  %table.table.table-striped.table-hover.old-table
+    %thead
+      %tr
+        %th.centered= OrderDetail.human_attribute_name(:id)
+        %th.centered= text("users.reservations.actions")
+        %th= text("users.reservations.timespan")
+        %th= OrderDetail.human_attribute_name(:product)
+        %th.centered= OrderDetail.human_attribute_name(:status)
+        %th.currency= OrderDetail.human_attribute_name(:actual_total)
+
+    %tbody
+      - OrderDetailPresenter.wrap(@order_details).each do |order_detail|
+        - reservation_presenter = ReservationUserActionPresenter.new(controller, order_detail.reservation)
```

Comment:
I looks like we're only using the presenter for the edit link. Would it make sense to remove the use of the presenter and just use the path helper? Or really, do we need to make it link to the view/edit link on this admin view, and have them use the order detail popup for modifying times.

I think if we get rid of the use of the presenter here, we can clean up the additional conditionals in the presenter that check if it needs to be an admin link or not.